website/project/model.py

```diff
@@ -4290,15 +4292,30 @@ def _email_template_context(self, user, node, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        log_type = NodeLog.PROJECT_REGISTERED
+        submitted_time = None
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+            is_prereg = "Prereg" in draft.registration_schema.name
```

Comment:
This is a nice place for a `try/except/else`. Put the `try/except` around just the one line where you're expecting the exception, and then put the rest in the `else` clause.